guides/common/modules/proc_retrieving-live-metrics-using-cli.adoc

```diff
@@ -1,19 +1,24 @@
 [id='retrieving-live-metrics-using-cli_{context}']
-= Retrieving Live Metrics using CLI
+= Retrieving Live Metrics in the CLI
```

Comment:
Imho "using" was better here, but should be "using the"

Reply:
It actually should be "by using the CLI" for complete clarity, which is so damn long :D I was thinking about leaving it out altogether. What do you think about that?

Reply:
Yeah that works.